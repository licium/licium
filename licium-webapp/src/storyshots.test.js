import initStoryshots from '@storybook/addon-storyshots'
import moment from 'moment-timezone';

initStoryshots({

    /* configuration options */
})
