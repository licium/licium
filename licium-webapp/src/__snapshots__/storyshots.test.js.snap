// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots ActionButton Disabled 1`] = `
<div
  className="dropdown is-hoverable"
>
  <div
    className="dropdown-trigger"
  >
    <button
      aria-controls="dropdown-menu"
      aria-haspopup="true"
      className="button is-primary"
      disabled={true}
    >
      <span>
        Actions
      </span>
      <span
        className="icon is-small"
      >
        <i
          aria-hidden="true"
          className="fas fa-angle-down"
        />
      </span>
    </button>
  </div>
  <div
    className="dropdown-menu"
    role="menu"
  >
    <div
      className="dropdown-content"
    >
      <div
        className="dropdown-item"
      >
        <a
          href="/registration/"
          onClick={[Function]}
        >
          Registration
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots ActionButton Enabled 1`] = `
<div
  className="dropdown is-hoverable"
>
  <div
    className="dropdown-trigger"
  >
    <button
      aria-controls="dropdown-menu"
      aria-haspopup="true"
      className="button is-primary"
    >
      <span>
        Actions
      </span>
      <span
        className="icon is-small"
      >
        <i
          aria-hidden="true"
          className="fas fa-angle-down"
        />
      </span>
    </button>
  </div>
  <div
    className="dropdown-menu"
    role="menu"
  >
    <div
      className="dropdown-content"
    >
      <div
        className="dropdown-item"
      >
        <a
          href="/registration/"
          onClick={[Function]}
        >
          Registration
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots AppHeader Logo 1`] = `
<div
  className="app-header"
>
  <a
    href="/"
  >
    <svg
      className="sm"
    >
      Logo.svg
    </svg>
  </a>
  licium
</div>
`;

exports[`Storyshots AssetPanel Default 1`] = `
<nav
  className="asset-panel panel"
>
  <p
    className="panel-heading"
    style={
      Object {
        "display": "flex",
        "justifyContent": "space-between",
      }
    }
  >
    <span>
      0
       assets selected
    </span>
    <span>
      <button
        className="button is-light is-small "
        onClick={[Function]}
      >
        <i
          className="icon fas fa-caret-down"
        />
      </button>
    </span>
  </p>
  <div
    className="asset-list expanded"
  />
</nav>
`;

exports[`Storyshots AssetPanel With Selected Assets 1`] = `
<nav
  className="asset-panel panel"
>
  <p
    className="panel-heading"
    style={
      Object {
        "display": "flex",
        "justifyContent": "space-between",
      }
    }
  >
    <span>
      10
       assets selected
    </span>
    <span>
      <button
        className="button is-light is-small "
        onClick={[Function]}
      >
        <i
          className="icon fas fa-caret-down"
        />
      </button>
    </span>
  </p>
  <div
    className="asset-list expanded"
  >
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      hack_personal_loan_account.ots
    </a>
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      maximized.qxb
    </a>
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      end_to_end_target_adapter.ecelp7470
    </a>
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      impactful_plastic.appcache
    </a>
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      indexing.wmf
    </a>
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      rubber_generate.mov
    </a>
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      web_enabled.wml
    </a>
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      agp.pclxl
    </a>
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      deposit_kansas.rgb
    </a>
    <a
      className="panel-block"
    >
      <span
        className="panel-icon"
      >
        <i
          aria-hidden="true"
          className="fas fa-book"
        />
      </span>
      qatari_rial_withdrawal_tennessee.rdf
    </a>
  </div>
</nav>
`;

exports[`Storyshots Assets Default 1`] = `
<div
  className="container"
>
  <h2
    className="css-84al3w"
  >
    ISCC Entries
  </h2>
  <div
    className="level"
  >
    <div
      className="level-left"
    />
    <div
      className="level-right"
    >
      <div
        className="level-item"
      >
        <div
          className="dropdown is-hoverable"
        >
          <div
            className="dropdown-trigger"
          >
            <button
              aria-controls="dropdown-menu"
              aria-haspopup="true"
              className="button is-primary"
              disabled={true}
            >
              <span>
                Actions
              </span>
              <span
                className="icon is-small"
              >
                <i
                  aria-hidden="true"
                  className="fas fa-angle-down"
                />
              </span>
            </button>
          </div>
          <div
            className="dropdown-menu"
            role="menu"
          >
            <div
              className="dropdown-content"
            >
              <div
                className="dropdown-item"
              >
                <a
                  href="/registration/"
                  onClick={[Function]}
                >
                  Registration
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="level"
  >
    <table
      className="table is-bordered is-striped is-hoverable"
    >
      <thead>
        <tr>
          <th
            onClick={[Function]}
          >
            Title
          </th>
          <th
            onClick={[Function]}
          >
            Meta-Code
          </th>
          <th
            onClick={[Function]}
          >
            Content-Code
          </th>
          <th
            onClick={[Function]}
          >
            Filename
          </th>
          <th
            onClick={[Function]}
          >
            Location
          </th>
          <th
            onClick={[Function]}
          >
            Stamp
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            cambridgeshire.ait
          </td>
          <td>
            yxtih
          </td>
          <td>
            vw0fb
          </td>
          <td>
            hack_personal_loan_account.ots
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2019-10-06T02:12:31.178+00:00
          </td>
        </tr>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            latvian_lats.rip
          </td>
          <td>
            hzyob
          </td>
          <td>
            c7pii
          </td>
          <td>
            maximized.qxb
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2019-12-29T07:04:14.834+00:00
          </td>
        </tr>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            synergized_decentralized.x3dbz
          </td>
          <td>
            4on6s
          </td>
          <td>
            j22vm
          </td>
          <td>
            end_to_end_target_adapter.ecelp7470
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2020-07-11T09:03:17.073+00:00
          </td>
        </tr>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            shoes.txf
          </td>
          <td>
            9t99z
          </td>
          <td>
            twrn2
          </td>
          <td>
            impactful_plastic.appcache
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2020-04-08T16:57:21.893+00:00
          </td>
        </tr>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            invoice.bz
          </td>
          <td>
            qvsc0
          </td>
          <td>
            m0kt7
          </td>
          <td>
            indexing.wmf
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2019-09-20T17:57:24.339+00:00
          </td>
        </tr>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            loop_california_mexican_peso_mexican_unidad_de_inversion_(udi).ssdl
          </td>
          <td>
            gbsxr
          </td>
          <td>
            dfjeq
          </td>
          <td>
            rubber_generate.mov
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2019-12-23T13:00:17.314+00:00
          </td>
        </tr>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            payment.sxg
          </td>
          <td>
            fcwe1
          </td>
          <td>
            jnoq7
          </td>
          <td>
            web_enabled.wml
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2020-02-28T01:51:17.976+00:00
          </td>
        </tr>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            tasty_advanced_electronics.pki
          </td>
          <td>
            0aehn
          </td>
          <td>
            hhl5v
          </td>
          <td>
            agp.pclxl
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2019-12-29T09:36:01.979+00:00
          </td>
        </tr>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            vision_oriented_rand_loti.m1v
          </td>
          <td>
            rzvls
          </td>
          <td>
            rs4va
          </td>
          <td>
            deposit_kansas.rgb
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2020-04-07T13:12:52.188+00:00
          </td>
        </tr>
        <tr
          className=""
          onClick={[Function]}
        >
          <td>
            transmitter_liaison_silver.cb7
          </td>
          <td>
            st70p
          </td>
          <td>
            vfrff
          </td>
          <td>
            qatari_rial_withdrawal_tennessee.rdf
          </td>
          <td>
            d:/some/folder
          </td>
          <td>
            2020-01-31T12:48:37.821+00:00
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;

exports[`Storyshots ISCC Default 1`] = `
<div
  className="box iscc"
>
  <div
    className="message-header"
  >
    Title - Long
    <button
      className="delete"
      onClick={[Function]}
    />
  </div>
  <form
    className="form"
    onSubmit={[Function]}
  >
    <div
      className="columns"
    >
      <div
        className="column"
      >
        <div
          className="field"
        >
          <div
            className="control is-small"
          >
            <label
              className="label is-small"
            >
              Title
              :
            </label>
            <input
              className="input is-small"
              disabled={false}
              onChange={[Function]}
              required={true}
              type="text"
              value="Title - Long"
            />
          </div>
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="field"
        >
          <div
            className="control is-small"
          >
            <label
              className="label is-small"
            >
              Extra
              :
            </label>
            <input
              className="input is-small"
              disabled={false}
              onChange={[Function]}
              type="text"
              value="Extra Foto"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="field"
    >
      <input
        className="button is-small"
        type="submit"
        value="Regenerate Meta-ID"
      />
    </div>
    <div
      className="field"
    >
      <div
        className="control is-small"
      >
        <label
          className="label is-small"
        >
          ISCC
          :
        </label>
        <input
          className="input is-small"
          disabled={true}
          onChange={[Function]}
          type="text"
          value="CCm7y4abbVN6r-CYeLkPETE1TPr-CD9TdCywdGcK3-CRWGC2JciDeP5"
        />
      </div>
    </div>
    <div
      className="field"
    >
      <div
        className="control is-small"
      >
        <label
          className="label is-small"
        >
          Tophash
          :
        </label>
        <input
          className="input is-small"
          disabled={true}
          onChange={[Function]}
          type="text"
          value="aeed0934a90961e0fb6635e5c9bd13a4739c4c3f6e30aaffd75411d12e940316"
        />
      </div>
    </div>
    <div
      className="field"
    >
      <div
        className="control is-small"
      >
        <label
          className="label is-small"
        >
          GMT
          :
        </label>
        <input
          className="input is-small"
          disabled={true}
          onChange={[Function]}
          type="text"
          value="IMAGE"
        />
      </div>
    </div>
    <div
      className="field"
    >
      <div
        className="columns "
      >
        <div
          className="column is-narrow"
        >
          <div
            className="columns is-mobile"
          >
            <div
              className="column is-narrow"
            >
              <p
                className="title is-7"
              >
                Meta-ID
              </p>
              <p
                className="subtitle is-7"
              >
                CCm7y4abbVN6r
              </p>
              <p
                className="is-7"
              >
                <code>
                  1001011000101110
                </code>
                <br />
                <code>
                  1110111101010101
                </code>
                <br />
                <code>
                  0010010110010011
                </code>
                <br />
                <code>
                  1000100001011111
                </code>
              </p>
            </div>
            <div
              className="column is-narrow"
            >
              <p
                className="title is-7"
              >
                Content-ID
              </p>
              <p
                className="subtitle is-7"
              >
                CYeLkPETE1TPr
              </p>
              <p
                className="is-7"
              >
                <code>
                  1110000110000011
                </code>
                <br />
                <code>
                  0001100001100101
                </code>
                <br />
                <code>
                  1001111011001111
                </code>
                <br />
                <code>
                  1001110001101001
                </code>
              </p>
            </div>
          </div>
        </div>
        <div
          className="column is-narrow"
        >
          <div
            className="columns is-mobile"
          >
            <div
              className="column is-narrow"
            >
              <p
                className="title is-7"
              >
                Data-ID
              </p>
              <p
                className="subtitle is-7"
              >
                CD9TdCywdGcK3
              </p>
              <p
                className="is-7"
              >
                <code>
                  0011000110001010
                </code>
                <br />
                <code>
                  1100101011001100
                </code>
                <br />
                <code>
                  0101100110110111
                </code>
                <br />
                <code>
                  1110001101001100
                </code>
              </p>
            </div>
            <div
              className="column is-narrow"
            >
              <p
                className="title is-7"
              >
                Data-ID
              </p>
              <p
                className="subtitle is-7"
              >
                CRWGC2JciDeP5
              </p>
              <p
                className="is-7"
              >
                <code>
                  1010111011101101
                </code>
                <br />
                <code>
                  0000100100110100
                </code>
                <br />
                <code>
                  1010100100001001
                </code>
                <br />
                <code>
                  0110000111100000
                </code>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </form>
</div>
`;

exports[`Storyshots ISCCField Default 1`] = `
<div
  className="field"
>
  <div
    className="control is-small"
  >
    <label
      className="label is-small"
    >
      Title
      :
    </label>
    <input
      className="input is-small"
      disabled={true}
      onChange={[Function]}
      type="text"
      value="A Title"
    />
  </div>
</div>
`;

exports[`Storyshots ISCCField Editable 1`] = `
<div
  className="field"
>
  <div
    className="control is-small"
  >
    <label
      className="label is-small"
    >
      Title
      :
    </label>
    <input
      className="input is-small"
      disabled={false}
      onChange={[Function]}
      type="text"
      value="A Title"
    />
  </div>
</div>
`;

exports[`Storyshots ISCCRegistration Default 1`] = `
<div
  className="container"
>
  <div
    className="field"
  >
    <label
      className="label"
    >
      Generate from file
    </label>
    <section>
      <div
        className="dropzone"
        onBlur={[Function]}
        onClick={[Function]}
        onDragEnter={[Function]}
        onDragLeave={[Function]}
        onDragOver={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        tabIndex={0}
      >
        <input
          autoComplete="off"
          multiple={true}
          onChange={[Function]}
          onClick={[Function]}
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
          type="file"
        />
        <p>
          <span
            className="icon"
          >
            <i
              className="fas fa-download"
            />
          </span>
        </p>
        <p>
          Drag 'n' drop some files here, or click to select files
        </p>
      </div>
    </section>
  </div>
  
</div>
`;

exports[`Storyshots Identification Table Empty Table 1`] = `
<table
  className="table is-bordered is-striped is-hoverable"
>
  <thead>
    <tr>
      <th
        onClick={[Function]}
      >
        Title
      </th>
      <th
        onClick={[Function]}
      >
        Meta-Code
      </th>
      <th
        onClick={[Function]}
      >
        Content-Code
      </th>
      <th
        onClick={[Function]}
      >
        Filename
      </th>
      <th
        onClick={[Function]}
      >
        Location
      </th>
      <th
        onClick={[Function]}
      >
        Stamp
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;

exports[`Storyshots Identification Table With Entries 1`] = `
<table
  className="table is-bordered is-striped is-hoverable"
>
  <thead>
    <tr>
      <th
        onClick={[Function]}
      >
        Title
      </th>
      <th
        onClick={[Function]}
      >
        Meta-Code
      </th>
      <th
        onClick={[Function]}
      >
        Content-Code
      </th>
      <th
        onClick={[Function]}
      >
        Filename
      </th>
      <th
        onClick={[Function]}
      >
        Location
      </th>
      <th
        onClick={[Function]}
      >
        Stamp
      </th>
    </tr>
  </thead>
  <tbody>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        cambridgeshire.ait
      </td>
      <td>
        yxtih
      </td>
      <td>
        vw0fb
      </td>
      <td>
        hack_personal_loan_account.ots
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2019-10-06T02:12:31.178+00:00
      </td>
    </tr>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        latvian_lats.rip
      </td>
      <td>
        hzyob
      </td>
      <td>
        c7pii
      </td>
      <td>
        maximized.qxb
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2019-12-29T07:04:14.834+00:00
      </td>
    </tr>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        synergized_decentralized.x3dbz
      </td>
      <td>
        4on6s
      </td>
      <td>
        j22vm
      </td>
      <td>
        end_to_end_target_adapter.ecelp7470
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2020-07-11T09:03:17.073+00:00
      </td>
    </tr>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        shoes.txf
      </td>
      <td>
        9t99z
      </td>
      <td>
        twrn2
      </td>
      <td>
        impactful_plastic.appcache
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2020-04-08T16:57:21.893+00:00
      </td>
    </tr>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        invoice.bz
      </td>
      <td>
        qvsc0
      </td>
      <td>
        m0kt7
      </td>
      <td>
        indexing.wmf
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2019-09-20T17:57:24.339+00:00
      </td>
    </tr>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        loop_california_mexican_peso_mexican_unidad_de_inversion_(udi).ssdl
      </td>
      <td>
        gbsxr
      </td>
      <td>
        dfjeq
      </td>
      <td>
        rubber_generate.mov
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2019-12-23T13:00:17.314+00:00
      </td>
    </tr>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        payment.sxg
      </td>
      <td>
        fcwe1
      </td>
      <td>
        jnoq7
      </td>
      <td>
        web_enabled.wml
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2020-02-28T01:51:17.976+00:00
      </td>
    </tr>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        tasty_advanced_electronics.pki
      </td>
      <td>
        0aehn
      </td>
      <td>
        hhl5v
      </td>
      <td>
        agp.pclxl
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2019-12-29T09:36:01.979+00:00
      </td>
    </tr>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        vision_oriented_rand_loti.m1v
      </td>
      <td>
        rzvls
      </td>
      <td>
        rs4va
      </td>
      <td>
        deposit_kansas.rgb
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2020-04-07T13:12:52.188+00:00
      </td>
    </tr>
    <tr
      className=""
      onClick={[Function]}
    >
      <td>
        transmitter_liaison_silver.cb7
      </td>
      <td>
        st70p
      </td>
      <td>
        vfrff
      </td>
      <td>
        qatari_rial_withdrawal_tennessee.rdf
      </td>
      <td>
        d:/some/folder
      </td>
      <td>
        2020-01-31T12:48:37.821+00:00
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`Storyshots Logo Large 1`] = `
<svg
  className="l"
>
  Logo.svg
</svg>
`;

exports[`Storyshots Logo Medium 1`] = `
<svg
  className="m"
>
  Logo.svg
</svg>
`;

exports[`Storyshots Logo Original Size 1`] = `
<svg>
  Logo.svg
</svg>
`;

exports[`Storyshots Logo Small 1`] = `
<svg
  className="sm"
>
  Logo.svg
</svg>
`;

exports[`Storyshots Menu Default 1`] = `
<div
  className="css-1domaf0"
>
  <div
    className="css-j7qwjs"
  >
    <section>
      <div
        onBlur={[Function]}
        onClick={[Function]}
        onDragEnter={[Function]}
        onDragLeave={[Function]}
        onDragOver={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        tabIndex={0}
      >
        <input
          autoComplete="off"
          multiple={true}
          onChange={[Function]}
          onClick={[Function]}
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
          type="file"
        />
        <button
          aria-disabled={false}
          className="css-z70eed"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Generate ISCCs
        </button>
      </div>
    </section>
    <a
      download="filename.json"
      href="data:text/json;charset=utf-8,undefined"
    >
      Download Json
    </a>
  </div>
</div>
`;

exports[`Storyshots Registration success Default 1`] = `
<section
  className="hero is-medium is-success"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Registration successful
      </h1>
      <h2
        className="subtitle"
      >
        You sucessfully registered 
        0
         assets.
      </h2>
      <a
        href="/"
        onClick={[Function]}
      >
        <button
          className="button"
        >
          Back to home
        </button>
      </a>
    </div>
  </div>
</section>
`;

exports[`Storyshots Welcome Page Default 1`] = `
<section
  className="hero is-medium is-primary"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Welcome to licium.
      </h1>
      <h2
        className="subtitle"
      >
        Licium does cool stuff with the blockchain.
      </h2>
      <a
        href="/assets"
        onClick={[Function]}
      >
        <button
          className="button "
        >
          Show your assets
        </button>
      </a>
      <a
        href="/iscc-registration"
        onClick={[Function]}
      >
        <button
          className="button "
        >
          Generate ISCC
        </button>
      </a>
    </div>
  </div>
</section>
`;
